.gitignore

```diff
@@ -20,3 +20,7 @@ Session.vim
 *.orig
 *.patch
 *.rej
+
+# goland, vscode
```

Comment:
can you please update the comment like `IDE related files (golang,vscode)`

Reply:
oh, sorry about detail... done!